backend/core/security.py

```
@@ -3,6 +3,8 @@
 from datetime import datetime, timedelta, timezone
 import os
 from dotenv import load_dotenv
+import pyotp
+from datetime import datetime, timedelta
```

Ruff (0.8.2)

Redefinition of unused `datetime` from line 3

Remove definition: `datetime`

(F811)